docs/dev/apis/lethallib/customscrap.md

```diff
@@ -0,0 +1,94 @@
+
+## Item Data
+The first thing you should create is the Item Data for your custom scrap, which can be created right-clicking in your asset window and going to create -> ScriptableObject -> Item
+![Image of the context menu in the asset browser of Unity, directing to Create -> ScriptableObjects -> Item](/images/lethallib/customscrap/CreateItem.png)
+
+This item data lets you configure basically everything about your item. There are a lot of options, but the important ones for scrap are:
+- **Item Name**: the name of the item.
+- **Spawn Position Types**: Spawn position types this item can spawn at. If the list is empty, the scrap can spawn anywhere (recommended to just leave this empty except for stuff like special scrap for custom moons).
+- **Two Handed**: Whether the item gives the "Hands Full" message and prevents changing items.
+- **Two Handed Animation**: Whether the item uses the two-handed animation for carrying or not.
+- **Weight**: A somewhat arbitrary number determing how heavy an item is. For reference, the Brush scrap is weight 1.1, the Cash Register is 1.8; pick a number that feels right.
+- **Item Spawns On Ground**: Should be true for scrap.
+- **Highest Sale Percentage**: Unknown; All items have this set to 80. Set it to that for safety.
+- **Is Conductive Metal**: Whether this item attracts lightning.
+- **Max/Min Value**: The value range for the item. Does not directly correlate to the range in game; a scalar is applied based on the moon it spawns on. For reference, the Big Bolt uses the range Max-80, Min-50.
+- **Spawn Prefab**: The prefab representing the item in the world. Creation of this is detailed later in this guide.
+- **Rotation/Position/Vertical Offset**: The offset values for holding the item in your hand. Mostly needs manual fiddling with until it looks right for a given item.
+- **Grab SFX, Drop SFX, Item Icon**: These must be assigned for the scrap to fully function.
+- **Allow Dropping Ahead of Player**: Whether the item can be dropped in front of the player, as opposed to only dropping directly below their feet.
+- **Is Defensive Weapon**: For scrap that also acts as weapons; requires more special scripting, so for most cases should be left off.
+
+Everything else is optional, has no effect on scrap, or shouldn't be modified (e.g. Netcode variables should be left alone).
+
+## Item Prefab
+An item prefab needs to have a specific setup to work correctly with all the Lethal Company systems. The prefab should have only the root object and a child object called "ScanNode":
+![Image of the Big Bolt scrap prefab structure.](/images/lethallib/customscrap/ScrapPrefab.png)
+
+### Root Object
+The root object should have the following:
+- Mesh Renderer + Mesh Filter for your model
+- Box collider roughly encompassing your model
+- Physics Prop script (see below for configuration)
+- Audio Source with no clip assigned (see below for output setup)
+- Network Object (with default checkbox configuration)
```

Comment:
False and very important. the network object must have auto object parent sync turned off or remote clients will break and raise many exceptions when picking up. Dont destroy with owner should also be checked I believe.